src/sidebar/components/annotation-action-bar.js

```diff
@@ -10,9 +10,22 @@ import { withServices } from '../util/service-context';
 import AnnotationShareControl from './annotation-share-control';
 import Button from './button';
 
+/** @typedef {import("../../types/api").Annotation} Annotation */
+
+/**
+ * @typedef AnnotationActionBarProps
+ * @prop {Annotation} annotation - The annotation in question
+ * @prop {(Event) => any} onReply - Callbacks for when action buttons are clicked/tapped
+ * @prop {Object} annotationsService
+ * @prop {Object} settings
```

Comment:
`settings` could use the `HostConfig` type. See the definition of the `isShareable` function.

Reply:
thanks!